views/news/news-container.js

```diff
@@ -43,6 +44,18 @@ export default class NewsContainer extends React.Component {
     this.refresh()
   }
 
+  findImage = (content: string) => {
+    let re = '<img.*?src="([^"]*)"[^>]*>(?:</img>)?'
+    let imageMatch = content.match(re)
+
+    if (imageMatch !== null && imageMatch !== undefined) {
+      let src = imageMatch[1]
+      return <Image source={{uri: src}} style={styles.itemImage} />
+    }
+
+    return false
```

Comment:
Generally, an optional react component is denoted with `null` for the no-component case, not `false`. Please switch this to use `null`.